decorator_trimmer.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator_base'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..9]
+  end
+end
```

Comment:
- [ ] the name should have a maximum of 10 characters. Kindly, make sure that your TrimeDecorator class look as the snippet code below to ensure that you meet the requirement:
```ruby
def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end
```
Make sure that you complete the missing syntax!